pandas/core/groupby/groupby.py

```diff
@@ -598,14 +598,7 @@ def pipe(self, func, *args, **kwargs):
     plot = property(GroupByPlot)
 
     def _make_wrapper(self, name):
-        if name not in self._apply_whitelist:
```

Comment:
Nice cleanup. At a glance this looks like a pretty interesting factory...

Reply:
Yah, this used to be called from within `__gettattr__` (and `exec` was involved)  Recently changed to be called from properties generated directly from apply_whitelist, so this check is no longer necessary